cypher-editor-support/test/autocompletion/parameter.test.js

```diff
@@ -26,6 +26,14 @@ describe('AutoCompletion - Parameter', () => {
     it('yields parameter name', () => {
       checkCompletionTypes('RETURN $▼a', true, [{ type: CompletionTypes.PARAMETER }]);
     });
+
+    it('yields parameter name w/o first char typed', () => {
```

Comment:
There are multiple tests in this file that are ignored (`xit` instead of `it` is used).

Unignore them and ensure that they are passing.